src/api_gateway/genReq.ts

```
@@ -0,0 +1,221 @@
+import * as globals from "../utility/globals.js";
+import checkFireWallBlocking from "./checkFireWallBlocking.js";
+
+const sleep = (ms) => new Promise((resolve) => setTimeout(resolve, ms));
+
+/**
+ * Given a URL, generates a new API Gateway for it and returns the response of the URL.
+ * @param {string} url The URL to generate an API Gateway for.
+ * @param {object} [headers] The headers to include in the request.
+ * @returns {Promise<string>} The response of the URL.
+ */
+const get = async (url: string, headers: {} = {}): Promise<string> => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)